MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -2,33 +2,56 @@
 // The .NET Foundation licenses this file to you under the MS-PL license.
 // See the LICENSE file in the project root for more information.
 
+using System.Threading;
+using System.Threading.Tasks;
 using MvvmCross.Exceptions;
 using MvvmCross.Logging;
 using MvvmCross.Navigation;
 
 namespace MvvmCross.ViewModels
 {
     public class MvxAppStart<TViewModel>
-        : IMvxAppStart
+        : IMvxAppStart, IMvxAppStartAsync
         where TViewModel : IMvxViewModel
     {
         protected readonly IMvxNavigationService NavigationService;
 
+        private int startHasCommenced;
+
         public MvxAppStart(IMvxNavigationService navigationService)
         {
             NavigationService = navigationService;
         }
 
-        public async void Start(object hint = null)
+
+
+        public void Start(object hint = null)
         {
+            // Check whether Start has commenced, and return if it has
+            if (Interlocked.CompareExchange(ref startHasCommenced, 1, 0) == 1) return;
+
             if (hint != null) {
                 MvxLog.Instance.Trace("Hint ignored in default MvxAppStart");
             }
+            
             try {
-                await NavigationService.Navigate<TViewModel>();
+                _startTaskNotifier = MvxNotifyTask.Create(async ()=> await NavigationService.Navigate<TViewModel>());
```

Comment:
The try/catch here won't work, as MvxNotifyTask doesn't re-throw exceptions. You can use the newly added `onException` callback for that.